docs/handbook/third-party-plugins.rst

```diff
@@ -0,0 +1,18 @@
+Third Party Plugins
+===================
+
+Pillow uses a plugin model which allows users to add their own
+decoders and encoders to the library, without any changes to the library
+itself.
+
+Here is a list of PyPI projects that offer additional plugins:
+
+    * :pypi:`DjvuRleImagePlugin`: Plugin for the DjVu RLE image format as defined in the DjVuLibre docs.
+    * :pypi:`heif-image-plugin`: Simple HEIF/HEIC images plugin, based on the pyheif library.
+    * :pypi:`jxlpy`: Introduces reading and writing support for JPEG XL.
+    * :pypi:`pillow-heif`: Python bindings to libheif for working with HEIF images.
+    * :pypi:`pillow-jpls`: Plugin for the JPEG-LS codec, based on the Charls JPEG-LS implemetation. Python bindings implemented using pybind11.
+    * :pypi:`pillow-jxl-plugin`: Plugin for JPEG-XL, using Rust for bindings.
+    * :pypi:`pillow-mbm`: Adds support for KSP's proprietary MBM texture format.
+    * :pypi:`pillow-svg`: Implements basic SVG read support. Currently supports basic paths, shapes, and text.
+    * :pypi:`raw-pillow-opener`: Simple camera raw opener, based on the rawpy library.
```

Comment:
Dedent so not quoted:

<img width="772" alt="image" src="https://github.com/user-attachments/assets/f159579d-5e82-4b4b-96b8-f62663d051e1" />


https://pillow--8910.org.readthedocs.build/en/8910/handbook/third-party-plugins.html

Reply:
I've pushed a commit. And created https://github.com/python-pillow/Pillow/pull/8915